src/main/java/org/folio/rest/impl/OaiPmhImpl.java

```diff
@@ -33,6 +33,7 @@
 
 import javax.ws.rs.core.Response;
 
+import io.vertx.core.*;
```

Comment:
Don't use the * import

Reply:
I just forgot to setup the code format settings at my laptop) I've used it to work for last days. Thank you for note. Fixed.